install-panel.sh

```diff
@@ -787,8 +813,12 @@ letsencrypt() {
   # Obtain certificate
   certbot --nginx --redirect --no-eff-email --email "$email" -d "$FQDN" || FAILED=true
 
+  if [ "$CONFIGURE_SSL_PMA" == true ]; then
+    certbot --nginx --redirect --no-eff-email --email "$email" -d "$PMA_FQDN" || FAILED=true
+  fi
+
   # Check if it succeded
-  if [ ! -d "/etc/letsencrypt/live/$FQDN/" ] || [ "$FAILED" == true ]; then
+  if [ ! -d "/etc/letsencrypt/live/$FQDN/" ] && [ ! -d "/etc/letsencrypt/live/$PMA_FQDN/" ] || [ "$FAILED" == true ]; then
```

Comment:
That will not work as it would fail, if the user isn't installing PMA and using letsencrypt.